tests/components/climate/common.py

```diff
@@ -0,0 +1,111 @@
+"""Collection of helper methods."""
```

Comment:
Can we update the description of each of these files to say that these are legacy helpers and should not be mimicked in new components.

Reply:
:+1: I've updated the descriptions.